src/main/plugins/org.dita.html5/xsl/topic.xsl

```diff
@@ -1247,8 +1257,9 @@ See the accompanying LICENSE file for applicable license.
     <xsl:apply-templates select="*[contains(@class, ' ditaot-d/ditaval-startprop ')]" mode="out-of-line"/>
     <xsl:call-template name="spec-title-nospace"/>
     <pre>
-      <xsl:attribute name="class" select="name()"/>
-      <xsl:call-template name="commonattributes"/>
+      <xsl:call-template name="commonattributes">
+        <xsl:with-param name="default-output-class" select="name()"/>
```

Comment:
This seems odd ... normal behavior with the old one would be set the class, then it's basically ignored by the `commonattributes`, which includes the element name by default. Now we're calling common attributes, but passing in a default of the element name, which ... should match default processing?